gen_token.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This script allows external users to generate a temporary Github access token
+# that allows them to download Lekko's developer tools.
+# This script should be executed in the following context:
+#
+#   export LEKKO_APIKEY=lekko_************
+#   export HOMEBREW_GITHUB_API_TOKEN=$(./gen_token.sh)
```

Comment:
hmm the ergonomic way would be to `$(curl https://github.....<path-to-script> | sh)` but as I mentioned people should bristle at this: https://0x46.net/thoughts/2019/04/27/piping-curl-to-shell/

Maybe its two steps & we ask people to inspect?
```
$ curl https://github.com/<path-to-script> > gen_token.sh
# please inspect the output here!
$ export HOMEBREW_GITHUB_API_TOKEN=$(./gen_token.sh)
```
Or maybe since this is Alpha we just do option 1 since people may trust us

Reply:
discussed offline - the actual instructions will be 
```bash
export LEKKO_APIKEY=lekko_************
brew tap lekkodev/lekko
export HOMEBREW_GITHUB_API_TOKEN=$(brew --repo)/Library/Taps/lekkodev/homebrew-lekko/gen_token.sh
brew install lekko # or brew upgrade lekko
```